src/common/types/game_manager.ts

```diff
@@ -76,6 +76,7 @@ export interface GameManager {
 }
 
 export interface LibraryManager {
+  init?: () => Promise<void>
```

Comment:
I think requiring this `init` method and just making it do nothing for sideloads would be easier here
Coincidentally I ended up making this same change in #4889

Reply:
init is now required.